Sources/MongoSwift/Operations/ListCollectionsOperation.swift

```diff
@@ -76,9 +76,16 @@ public struct ListCollectionsOptions: Codable {
     /// The batchSize for the returned cursor.
     public var batchSize: Int?
 
+    /// Specifies whether to only return collections for which the user has privileges. Only has
+    /// an effect when used with `listCollectionNames`. See
+    /// [here](https://www.mongodb.com/docs/manual/reference/command/listCollections/)
```

Comment:
usually for links, we use `SeeAlso`, like this: https://github.com/mongodb/mongo-swift-driver/blob/56bc2e608ff4620e6141f64b26b247f3ad65ab59/Sources/MongoSwift/MongoConnectionString.swift#L13

Reply:
Thanks for the info
Updated `authorizedCollections` comment, omitted `Nil` parameters,  and used the `SeeAlso` tag. Prior commits resolved alphabetical order issues and a questionable test case that did not close properly

Reply:
hey, that's my test case you're talking about! ... jk, yeah the lack of cleanup was not great, thanks for fixing that 🙂